suites/pacific/rados/tier-2_rados_stretch_mode_upgrade.yaml

```diff
@@ -1,4 +1,10 @@
 # Suite contains tests related to election strategy and Stretched mode
```

Comment:
Can we rename this file to follow guidelines? 
tier-2_rados_test-strech-mode-upgrade.yaml

Reply:
Updated the file name according to guidedlines.